pyflask/manageNeuroconv/manage_neuroconv.py

```diff
@@ -69,6 +73,35 @@ def get_metadata_schema(source_data: Dict[str, dict]) -> Dict[str, dict]:
     return json.loads(json.dumps(dict(results=metadata, schema=schema), cls=NWBMetaDataEncoder))
 
 
+class objectview(object):
+    def __init__(self, d):
+        self.__dict__ = d
+
+
+def validate_metadata(parent: dict, function: str) -> dict:
+    """
+    Function used to validate data using an arbitrary NWB Inspector function
+    """
+    fn = nwbinspector.__dict__.get(function)
```

Comment:
```suggestion
    nwbfile_metadata = nwbfile_metadata or dict(session_description="This is a description",
            identifier="00001",
            session_start_time=datetime.now(),
    )
    fn = nwbinspector.__dict__.get(function)
```

Suggested logical design change - even the subject-level checks are technically run upon the NWBFile object, so you will always need to make a 'placeholder' NWBFile object

Only different with the Subject checks is they will also make a `Subject` that gets added to that placeholder NWB file